master/pkg/syncx/mapx/mapx.go

```diff
@@ -55,3 +55,23 @@ func (m *Map[K, V]) WithLock(f func(m map[K]V)) {
 
 	f(m.inner)
 }
+
+// Clear the map.
+func (m *Map[K, V]) Clear() {
+	m.mu.Lock()
+	defer m.mu.Unlock()
+	for k := range m.inner {
+		delete(m.inner, k)
+	}
```

Comment:
if you wait a day or two, you can just use https://tip.golang.org/ref/spec#Clear

Reply:
(gotta get go1.21)